jll/T/Tesseract_jll/Compat.toml

```diff
@@ -7,5 +7,6 @@ Libtiff_jll = "4.3.0-4.4"
 ["4.1.100-5"]
 Cairo_jll = "1.16.1-1"
 JLLWrappers = "1.2.0-1"
+Leptonica_jll = "1.82.0-1.82"
```

Comment:
It has to be under the 4-5 section: https://github.com/JuliaRegistries/General/blob/6f8f32827dc285ce3e518b3e8920d41ded94dd6b/jll/T/Tesseract_jll/Deps.toml#L1-L6

Reply:
Thanks!